tests/test_bqm.py

```diff
@@ -3175,3 +3175,21 @@ def test_spin(self, name, BQM):
 
             self.assertAlmostEqual(bqm.energy(sample),
                                    sum(sample[v]*b for v, b in terms)**2)
+
+
+class TestAddBQM(unittest.TestCase):
+    @parameterized.expand(itertools.product(BQMs.items(), BQMs.items()))
```

Comment:
Doesn't actually matter, but you don't need the `BQM_info0` if you just do `itertools.product(BQMs.values(), repeat=2)`